WcaOnRails/app/models/competition.rb

```diff
@@ -512,7 +512,7 @@ def has_event?(event)
   end
 
   def has_entry_fee?
```

Comment:
This probably needs to be split into two separate functions now, and pick the right one for the three uses.

`has_entry_fee?` - Does the competition itself have a fee?

`has_fees?` - Any fees at all?